### ISSUE TITLE
关于代码中的loss

### ISSUE THREAD

**Issue by LinPeiMing** [FVL2020/SC-NAFSSR#3]
您好：
loss += 0.0025 * (loss_photo + 0.1 * loss_smooth + loss_cycle)
这个0.0025是什么意思呢？好像并没有在论文中读到诶，是用来监督SCAM的公式4么？那为什么是0.0025呢？
还有个小问题就是看代码里用的是 num_blks = 32，width = 32，而把 num_blks = 128 width = 128注释掉了？但论文说用的是NAFSSR-L，有点没看懂最后用的哪个。

**Comment by qiuzidian** [comment]
第一个问题应该是之前在实验里根据SCAM的个数调整过损失权重遗留的问题，但这个数值并不是很重要，保持在0.01左右就行。第二个问题你看的那几行代码应该是测试网络的代码，可以看option里的配置文件就是用的NAFSSR-L
